web/static/js/color-marker.js

```diff
@@ -20,11 +20,9 @@ class ColorMarker {
 
     if (!selector) return _this;
 
-    _this.$el = $(selector);
+    _this.el = document.querySelectorAll(selector)[0];
```

Comment:
Have you thought about using `#querySelector()` since you're just grabbing the first one?